plugins/dac_data_manager.c

```diff
@@ -841,7 +841,11 @@ static GtkWidget *gui_dds_mode_chooser_create(struct dds_tx *tx)
 	GtkWidget *box;
 	GtkWidget *dds_mode_lbl;
 	GtkComboBoxText *dds_mode;
-	bool no_buffer_support = !strcmp(tx->parent->name, "axi-ad9739a-hpc");
+	bool no_buffer_support=false;
+	if(!strcmp(tx->parent->name,"axi-ad9739a-lpc"))
+	  no_buffer_support = !strcmp(tx->parent->name, "axi-ad9739a-lpc");
```

Comment:
Well, you can directly set no_buffer_support to true. You already did the comparison in the if statement.
You can even go further and do something like:
if (name is axi-ad9739a-lpc or is axi-ad9739a-hpc) then set no_buffer_suport to true. Which I believe is more readable.

Reply:
Fixed